test/com/facebook/buck/features/go/testdata/cgo/src/mixed_with_c/BUCK.fixture

```diff
@@ -0,0 +1,22 @@
+go_binary(
+    name = "bin",
+    srcs = ["main.go"],
+    cgo = ":cgo",
+)
+
+cgo_library(
+    name = "cgo",
+    package_name = "main",
+    srcs = ["cgosrc.go"],
+    deps = [":lib"]
+)
+
+cxx_library(
+    name = "lib",
+    srcs = ["lib.c"],
+    exported_headers = ["lib.h"],
+    header_namespace = "",
```

Comment:
is the header_namespace important here? does it work without it?

Reply:
No, unfortunately. `header_namespace = ""` allows the C compiler running from buck root to compile CGo generated C files with `include "lib.h"`. Otherwise, C compiler would try to find `lib.h` from its current directory (i.e., buck root), resulting a failure.

The `--I` flag in this fix only tells the CGo compiler to find `lib.h` from the target's base directory. It doesn't pass that message to the C compiler.